xbmc/cores/ExternalPlayer/ExternalPlayer.cpp

```diff
@@ -452,13 +452,12 @@ BOOL CExternalPlayer::ExecuteAppW32(const char* strPath, const char* strSwitches
     CloseHandle(m_processInfo.hProcess);
     m_processInfo.hProcess = 0;
   }
-
-  return ret;
+  return (ret == 0);
```

Comment:
This changes behaviour.
After replacing it with `return ret == TRUE;` it fixes https://trac.kodi.tv/ticket/17983

Reply:
great, you found the root cause